launcher/app.py

```diff
@@ -2,174 +2,17 @@
 
 import os
 from datetime import datetime
-from kivy.lang import Builder
+from kivy.lang import Builder, builder
```

Comment:
double import of Builder, and not sure why the `builder` module is imported as well.